app/callbacks.py

```diff
@@ -46,6 +47,10 @@
 TEMP_DIR = tempfile.mkdtemp()
 du.configure_upload(app, TEMP_DIR)
 
+DEMO_DATA_URL = (
```

Comment:
For this demo data, the default scoring cutoff `0.05` would give nothing, better to change it to 0 to make sure the demo can display something for the first try.